tests/cnf/ran/ztp/tests/ztp-argocd-hub-templating.go

```diff
@@ -65,11 +65,19 @@ var _ = Describe("ZTP Argo CD Hub Templating Tests", Label(tsparams.LabelArgoCdH
 	It("should report an error for using autoindent function where not allowed", reportxml.ID("54240"), func() {
 		setupHubTemplateTest(tsparams.ZtpTestPathTemplatingAutoIndent)
```

Comment:
We should check for the version before we get the test set up so that we're not updating argo cd, creating a cgu, etc. if we don't need it

Reply:
Fixed. Thank you.